tests/cypress/integration/5-AI-SiteGen-onboarding-flow/fork.cy.js

```diff
@@ -0,0 +1,81 @@
+// <reference types="Cypress" />
+
+import { AdminBarCheck, DarkBGCheck, LightBGChcek } from "../wp-module-support/siteGenBG.cy";
+
+describe( 'SiteGen Fork Step', function () {
+	before( () => {
+		cy.visit(
+			'wp-admin/?page=nfd-onboarding#/wp-setup/step/fork'
+		);
+	} );
+
+    it( 'Check for the header admin bar', () => {
+        AdminBarCheck();
+    } );
+
+    it( 'Check for the existing dark background', () => {
+		DarkBGCheck();
+	} );
+
+    it( 'Check for the light background', () => {
+        LightBGChcek();
+    } );
+
+    it( 'Check for the heading and the title', () => {
+        cy.get( '.nfd-onboarding-step__heading__title' )
+            .should('be.visible')
+            .should('have.text', 'Welcome to WordPress');
+    } );
+
+    it ( 'Check for the subheading', () => {
+        cy.get( '.nfd-onboarding-step__heading__subtitle' ).should('be.visible');
+    } );
+
+    const OptionsDetails = (className,textValue,optionsValue) => {
+        cy.get(className)
+            .eq(optionsValue)
+            .find('.nfd-onboarding-sitegen-options__container__heading__title')
+            .invoke( 'text' )
+            .should('contain', textValue);
+        if(optionsValue!=2){   // Excluding the Last Option as it takes to new tab, just validating the title text
+            cy.get(className)
+                .eq(optionsValue)
+                .click();
+        };
+    };
+
+    it( 'Check for selection of different container options', () => {
+        let options = 0;
+        const className = '.nfd-onboarding-sitegen-options__container__options';
+        const arr = cy.get( className );
+		arr.each( () => {
```

Comment:
Can you check if the element count is always 3? Meaning there are only 3 tiles?

Reply:
This is not required. As, we'll sometimes have only two tiles available when the user doesn't have the AI site capabilities. So, it's better not to check the exact number.

Reply:
Added a check to validate the number of container options avaialable.